packages/carbon-web-components/src/components/tabs/tabs.ts

```diff
@@ -411,42 +411,64 @@ export default class CDSTabs extends HostListenerMixin(CDSContentSwitcher) {
     }
   }
 
-  render() {
+  /**
+   * Render the previous button if tablist is wider than container.
+   */
+  protected renderPreviousButton() {
     const {
       _isIntersectionLeftTrackerInContent: isIntersectionLeftTrackerInContent,
+    } = this;
+
+    return isIntersectionLeftTrackerInContent
+      ? html`
+          <button
+            part="prev-button"
+            tabindex="-1"
+            aria-hidden="true"
+            class="${prefix}--tab--overflow-nav-button ${prefix}--tabs__nav-caret-left ${prefix}--tab--overflow-nav-button--previous"
+            @click=${(_) =>
+            this._handleScrollButtonClick(_, {
+              direction: NAVIGATION_DIRECTION.Left,
+            })}>
+            ${ChevronLeft16()}
+          </button>
+        `
+      : null;
+  }
+
+  /**
+   * Render the next button if tablist is wider than container.
+   */
+  protected renderNextButton() {
+    const {
       _isIntersectionRightTrackerInContent: isIntersectionRightTrackerInContent,
+    } = this;
+
+    return isIntersectionRightTrackerInContent
+      ? html`
+          <button
+            part="next-button"
+            tabindex="-1"
+            aria-hidden="true"
+            class="${prefix}--tab--overflow-nav-button ${prefix}--tabs__nav-caret-right ${prefix}--tab--overflow-nav-button--next"
+            @click=${(_) =>
+              this._handleScrollButtonClick(_, {
+                direction: NAVIGATION_DIRECTION.Right,
+              })}>
+            ${ChevronRight16()}
+          </button>
+        `
+      : null;
+  }
+
+  render() {
+    const {
       _assistiveStatusText: assistiveStatusText,
       _handleSlotchange: handleSlotchange,
     } = this;
 
-    const previousButtonClasses = classMap({
-      [`${prefix}--tab--overflow-nav-button`]: true,
-      [`${prefix}--tabs__nav-caret-left`]: true,
-      [`${prefix}--tab--overflow-nav-button--previous`]: true,
-      [`${prefix}--tab--overflow-nav-button--hidden`]:
-        isIntersectionLeftTrackerInContent,
-    });
-    const nextButtonClasses = classMap({
-      [`${prefix}--tab--overflow-nav-button`]: true,
-      [`${prefix}--tabs__nav-caret-right`]: true,
-      [`${prefix}--tab--overflow-nav-button--next`]: true,
-      [`${prefix}--tab--overflow-nav-button--hidden`]:
```

Comment:
Simlar note to `previousButtonClasses` note.

Reply:
Same answer 🙂